Documentation/CodeSnippets/TypoScriptSyntax/CodeBlock3.rst.txt

```diff
@@ -1,15 +1,14 @@
 .. Automatic screenshot: Remove this line if you want to manually change this file
 
 .. code-block:: typoscript
-   :caption: Extension examples, file Configuration/TypoScript/Syntax/CodeBlock2/setup.typoscript
+   :caption: Extension examples, file Configuration/TypoScript/Syntax/CodeBlock3/setup.typoscript
 
-   myObject = TEXT
-   myObject {
+   myIdentifier = TEXT
+   myIdentifier {
```

Comment:
actually I liked "myObject" as it's slightly more descriptive

Reply:
We don't really have a rule for that. But not beeing an OOP language in TypoScript there are really no objects

Reply:
i removed the wording 'object' everywhere.

Reply:
I'm waiting for questions like "what I have to take for 'myIdentifier'" ;-)